motionplan/constraint.go

```diff
@@ -502,23 +502,71 @@ type LinearConstraint struct {
 	OrientationToleranceDegs float64
 }
 
+// GetLineToleranceMm returns the LineToleranceMm field in a LinearConstraint object.
+func (x *LinearConstraint) GetLineToleranceMm() float64 {
+	if x != nil {
+		return x.LineToleranceMm
+	}
+	return 0
+}
+
+// GetOrientationToleranceDegs returns the OrientationToleranceDegs field in a LinearConstraint object.
+func (x *LinearConstraint) GetOrientationToleranceDegs() float64 {
```

Comment:
Lets delete these functions.  Since the LinearConstraint has public fields we can just access them without a getter function, so no need to bloat the codebase with these.

Actually by the looks of it this file doesn't actually need any changes since all the code being added does similar things to these functions!